crates/accelerate/src/permutation.rs

```diff
@@ -111,10 +116,31 @@ fn _get_ordered_swap(py: Python, permutation_in: PyArrayLike1<i64>) -> PyResult<
     Ok(get_ordered_swap(&view).to_object(py))
 }
 
+#[pyfunction]
+#[pyo3(signature = (pattern))]
+fn _synth_permutation_basic(py: Python, pattern: PyArrayLike1<i64>) -> PyResult<CircuitData> {
+    let view = pattern.as_array();
+    let num_qubits = view.len();
+    Ok(CircuitData::from_standard_gates(
+        py,
+        num_qubits as u32,
+        get_ordered_swap(&view).iter().map(|(i, j)| {
```

Comment:
Is there a reason to leave this as a `pyfunction`? It seems more natural to have this take an `ArrayView1` or a slice and leave it as solely a private rust function.

Reply:
It's currently also used in the construction of the `PermutationGate` (and circuit, they are doubled atm). But I can change that code to instead use `synth_permutation_basic` which would be a bit better as it is actually part of the API as opposed to `_get_ordered_swap`.

Reply:
Done in b0fcf2a